satellite/identdb/identdb.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package identdb
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/storj"
+)
+
+// DB stores storagenode peer identities
+type DB interface {
```

Comment:
The package is called `identdb` and the full name ends up being `identdb.DB` which is weird.

Reply:
there is but followed existing name certdb-> identdb in Storj.io/storj/satellite/identdb

Reply:
The previous `certdb` wasn't great either. I spent some time thinking and it seems the most appropriate place for the interface would be in overlay package. It's storing after all critical storage node information and overlay is responsible for managing that information.

Hence let's move make the fully qualified named `overlay.Identities`, so the interface name would be `Identities`.

Reply:
PS: I'm not yet completely satisfied with the name `Identities`, but I think it should be good enough for this PR. `PeerIdentities` would be fine as well.